scripts/post-install.js

```diff
@@ -1,94 +1,3 @@
-#!/usr/bin/env node
```

Comment:
I think you need to preserve this line so that we can run `scripts/post-install.js` directly from the shell without prepending `node`.